meson.build

```diff
@@ -0,0 +1,27 @@
+project('tinyendian', 'd')
```

Comment:
On the [PR to D-YAML](https://github.com/dlang-community/D-YAML/pull/58/files#diff-969b60ad3d206fd45c208e266ccfed38R1), you also used the `meson_version` and `subproject_dir` parameters. Do we need them here?